integration_test/README.md

```diff
@@ -0,0 +1,17 @@
+This directory contains an integration test that confirms the basic functionality
+of the profiler works on GCE. In particular, this test confirms that the agent 
+can create and upload profiles from GCE, and that these profiles contain 
+symbolized samples from the benchmark application.
+
+More specifically, this test:
+1. Starts 3 GCE VMs, one to test node versions 6, 8, and 9. 
+Each GCE VM then:
+    1. Downloads the desired version of node, github, and build-essentials 
+    (the dependencies needed to run the test).
+    2. Clones the agent from github.
```

Comment:
"Clones the agent source code at the revision of interest" ?

Obviously it shouldn't use HEAD when testing PRs, so it should be stated somehow.

Reply:
Done.